eventmesh-webhook/eventmesh-webhook-admin/src/main/java/org/apache/eventmesh/webhook/admin/FileWebHookConfigOperation.java

```diff
@@ -49,77 +49,54 @@ public FileWebHookConfigOperation(String filePath) throws FileNotFoundException
 		}
-		return 1;
+		return writeToFile(webhookConfigFile, webHookConfig) ? 1 : 0;
 	}
 
 	@Override
 	public Integer updateWebHookConfig(WebHookConfig webHookConfig) {
-		String manufacturerName = webHookConfig.getManufacturerName();
-		String manuDirPath = filePath + FILE_SEPARATOR + manufacturerName;
-		File webhookConfigFile = new File(manuDirPath + FILE_SEPARATOR + webHookConfig.getManufacturerEventName() + FILE_EXTENSION);
+		File webhookConfigFile = new File(getWebhookConfigFilePath(webHookConfig));
 		if (!webhookConfigFile.exists()) {
-			logger.error("webhookConfig " + manufacturerName + "_" + webHookConfig.getManufacturerEventName() + " is not existed");
+			logger.error("webhookConfig " + webHookConfig.getCallbackPath() + " is not existed");
 			return 0;
 		}
-		try (FileWriter fw = new FileWriter(webhookConfigFile); BufferedWriter bw = new BufferedWriter(fw)) {
-			bw.write(JsonUtils.serialize(webHookConfig));
-		} catch (IOException e) {
-			e.printStackTrace();
-			return 0;
-		}
-		return 1;
+		return writeToFile(webhookConfigFile, webHookConfig) ? 1 : 0;
 	}
 
 	@Override
 	public Integer deleteWebHookConfig(WebHookConfig webHookConfig) {
-		String manufacturerName = webHookConfig.getManufacturerName();
-		String manuDirPath = filePath + FILE_SEPARATOR + manufacturerName;
-		File webhookConfigFile = new File(manuDirPath + FILE_SEPARATOR + webHookConfig.getManufacturerEventName() + FILE_EXTENSION);
+		File webhookConfigFile = new File(getWebhookConfigFilePath(webHookConfig));
 		if (!webhookConfigFile.exists()) {
-			logger.error("webhookConfig " + manufacturerName + "_" + webHookConfig.getManufacturerEventName() + " is not existed");
+			logger.error("webhookConfig " + webHookConfig.getCallbackPath() + " is not existed");
 			return 0;
 		}
 		return webhookConfigFile.delete() ? 1 : 0;
 	}
 
 	@Override
 	public WebHookConfig queryWebHookConfigById(WebHookConfig webHookConfig) {
-		String manufacturerName = webHookConfig.getManufacturerName();
-		String manuDirPath = filePath + FILE_SEPARATOR + manufacturerName;
-		File webhookConfigFile = new File(manuDirPath + FILE_SEPARATOR + webHookConfig.getManufacturerEventName() + FILE_EXTENSION);
+		File webhookConfigFile = new File(getWebhookConfigFilePath(webHookConfig));
 		if (!webhookConfigFile.exists()) {
-			logger.error("webhookConfig " + manufacturerName + "_" + webHookConfig.getManufacturerEventName() + " is not existed");
+			logger.error("webhookConfig " + webHookConfig.getCallbackPath() + " is not existed");
 			return null;
 		}
-
 		return getWebHookConfigFromFile(webhookConfigFile);
 	}
 
 	@Override
 	public List<WebHookConfig> queryWebHookConfigByManufacturer(WebHookConfig webHookConfig, Integer pageNum,
 			Integer pageSize) {
-		String manufacturerName = webHookConfig.getManufacturerName();
-		String manuDirPath = filePath + FILE_SEPARATOR + manufacturerName;
-		File manuDir = new File(manuDirPath);
+		File manuDir = new File(getWebhookConfigManuDir(webHookConfig));
```

Comment:
You don't need to call `getWebhookConfigManuDir` again
```suggestion
        String webHookFilePath = getWebhookConfigManuDir(webHookConfig);
		File manuDir = new File(webHookFilePath);
```